week-4/Homework/mandatory/4-groceries.js

```diff
@@ -28,19 +28,30 @@ Exercise 1:
   Then use console.log() to print out the list.
 */
 // Gather all week item names into this array
+
 let weeklyGroceriesToBuy = [];
+for (let property in weeklyMealPlan[property]) {
+  weeklyGroceriesToBuy.push(weeklyMealPlan);
+}
```

Comment:
inside this loop, weeklyMealPlan is the object and property are the keys so if you  weeklyMealPlan[property] you are getting the values. Needs to be fixed Laetitia